ga4gh/backend.py

```diff
@@ -192,6 +198,68 @@ def _getEnd(cls, variant):
         return variant.end
 
 
+class VariantAnnotationsIntervalIterator(IntervalIterator):
+    """
+    An interval iterator for annotations
+    """
+
+    def __init__(self, request, parentContainer):
+        super(VariantAnnotationsIntervalIterator, self).__init__(
+            request, parentContainer)
+        print('self._request.featureIds: ', self._request.featureIds)
+        if self._request.featureIds is None:
+            self._featureIds = []
+        else:
+            self._featureIds = set(self._request.featureIds)
+        self._effects = set(self._request.effects)
```

Comment:
It looks like the test apparatus you're using is setting default values. When I use the script you've posted the payload to the server looks like:

     {u'referenceId': u'', u'end': 70000, u'name': None, u'pageSize': None, u'pageToken': u'64792:0', u'start': 60000, u'effects': [], u'referenceName': u'1', u'featureIds': None, u'variantAnnotationSetId': u'YnJjYTE6MWtnUGhhc2UzLmNvcHk6dmFyaWFudGFubm90YXRpb25z'}

Try sending in a request directly to the endpoint, the request to VariantAnnotationsSearch should be allowed to be specify only an VariantAnnotationSetId.

    curl -i -H "Accept: application/json" -H "Content-Type: application/json" -X POST -d '{"variantAnnotationSetId": "YnJjYTE6MWtnUGhhc2UzOnZhcmlhbnRhbm5vdGF0aW9ucw=="}' http://localhost:8000/variantannotations/search